phalcon/mvc/model/criteriainterface.zep

```diff
@@ -192,6 +193,6 @@ interface CriteriaInterface
 	/**
 	 * Executes a find using the parameters built with the criteria
 	 */
-	public function execute() -> <ResultsetInterface>;
+	public function execute(<TransactionInterface> transaction = null) -> <ResultsetInterface>;
```

Comment:
The interface can't be changed for sure so you need to remove this change, not sure about others as i told. Theoretically if someone will extend Criteria and not override execute method everything will be fine. But if they have overrided execute method then it will be problem.